help/md/fnFreeK.md

```diff
@@ -1,9 +1,31 @@
 ## name
 fnFreeK
 ## title
+Free K Rate Matrix
 ## description
+This function generates and returns a free rates matrix. 
 ## details
+This function accepts both RealPos[] or RealPos[][] as the first argument to automatically
```

Comment:
The information on acceptable argument types is already given to us by the auto-generated portion of the help file. Here, it might be better to say that the first argument can be either a vector or a matrix of nonnegative real numbers, and explain the difference between these two kinds of input – i.e., when to supply a vector (and what the function does with it) and when to supply a matrix (and what the function does with it).